### ISSUE TITLE
Accessibility : Screen reader reads out of order

### ISSUE THREAD

**Issue by akhaderachana** [ngx-formly/ngx-formly#1606]
**Current behavior**
There is an accessibility issue using screen reader (NVDA, JAWS). The fields are not read in order by screen readers. Screen reader is reading fields alternately.

**Expected behavior**
Screen readers should read the fields in order

**Minimal reproduction of the problem with instructions**
https://gjjbbqdmmage.angular.stackblitz.io

In the above example, when a screen reader is reading the fields, it will read Name of Investment, then Stock Identifier, then Date of investment. (Instead of reading them in the order they appear on the screen).
This is when the screen reader is reading the entire page (not while tabbing into the fields).

**Comment by aitboudad** [comment]
Well, I don't have enough knowledge on that part so without some guidance it may difficult for me to help.
The provided example doesn't work could you please check it?

**Comment by akhaderachana** [comment]
Sorry about that. Can you try this -
https://stackblitz.com/angular/gjjbbqdmmage?file=src%2Fapp%2Fapp.component.ts

Just fyi, the screen reader might read all of the code in there. Thats why I had provided the above link with just the example.

I got this example from https://ngx-formly.github.io/ngx-formly/examples/advanced/repeating-section, if that helps.

**Comment by aitboudad** [comment]
I'm able to reproduce it now :+1:

**Comment by aitboudad** [comment]
setting display block to `formly-field` element solve the issue on my side:
```css
formly-field {
  display: block;
}
```

**Comment by akhaderachana** [comment]
This worked for me too.
Thanks a lot
